src/instructlab/cli/process/process_manager.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Union, cast
+import json
+
+# First Party
+from instructlab.defaults import DEFAULTS
+
+TASKS_FILE = (Path(DEFAULTS.PROCESS_DIR) / "process_tasks.json").expanduser()
```

Comment:
as discussed in the linked dev-doc, storing the processes in memory rather than on disk is a more long-term approach that will scale with various points of consumption  for ilab